h2o-docs/src/product/explainability.rst

```diff
@@ -0,0 +1,205 @@
+
+
+.. tabs::
+   .. code-tab:: r R
+
+        # Explain a model
+        exm <- h2o.explain(model, test)
+
+        # Explain an AutoML object
+        exa <- h2o.explain(aml, test)
+
+
+   .. code-tab:: python
+
+        # Explain a model
+        exm = model.explain(test)
+
+        # Explain an AutoML object
+        exa = aml.explain(test)
+
+
+
+
+Parameters
+~~~~~~~~~~
+
+TO DO: This was created looking at the R interface, may need some updates for Python.  
+
+- **object**: (R only) One of the following: an H2O model, a list of models, an H2OAutoML object, or an H2OAutoML leaderboard slice.
+
+- **test_frame**: An H2OFrame of test data.
+
+- **columns_of_interest**: A vector of column names. If specified, create plots only with these columns (where applicable).
+
+- **include_explanations**: If specified, do only the specified explanations. Mutually exclusive with exclude_explanations.
+
+- **exclude_explanations**: Exclude specified explanations.
+
+The available options (explainations) for ``include_explanations`` and ``exclude_explainations`` are:
+    
+    - ``"leaderboard"``
+    - ``"confusion_matrix"``
+    - ``"residual_analysis"``
+    - ``"variable_importance"``
+    - ``"variable_importance_heatmap"``
+    - ``"model_correlation_heatmap"``
+    - ``"shap_summary"``
+    - ``"pdp"``
+    - ``"ice"``
+
+- **top_n_features**: If ``columns_of_interest`` is missing, create plots only with the top n columns (where applicable).  Defaults to 5.
+
+- **best_of_family**: If True, explain only best of family models.
+
+- **user_overrides**: Overrides for individual explanations, e.g. ``list(shap_summary_plot = list(top_n_features = 50))`` in R. 
+
+Notes
+~~~~~
+
+TO DO.
+
+
+Code Examples
+-------------
+
+TO DO.
+
+Explain models
+~~~~~~~~~~~~~~
+
+Here’s an example showing basic usage of the ``h2o.explain()`` function in *R* and the ``explain()`` method in *Python*.  
+
+.. tabs::
+   .. code-tab:: r R
+
+        library(h2o)
+        h2o.init()
+
+        # Import a sample binary outcome train/test set into H2O
+        train <- h2o.importFile("https://s3.amazonaws.com/erin-data/higgs/higgs_train_10k.csv")
+        test <- h2o.importFile("https://s3.amazonaws.com/erin-data/higgs/higgs_test_5k.csv")
+
+        # Identify predictors and response
+        y <- "response"
+        x <- setdiff(names(train), y)
+
+        # For binary classification, response should be a factor
+        train[, y] <- as.factor(train[, y])
+        test[, y] <- as.factor(test[, y])
+
+        # Run AutoML for 20 base models
+        aml <- h2o.automl(x = x, y = y, 
+                          training_frame = train,
+                          max_models = 20,
+                          seed = 1)
+
+        # Explain all AutoML models                  
+        exa <- h2o.explain(aml, test)
```

Comment:
In R, the explanation has semantics similar to `ggplot2`, i.e., if we want to show it we have to use `print(exa)` or in case of Jupyter `repr(exa)`. In interactive use, it gets printed automatically since R REPL automatically prints, Rmarkdown also prints the value, and Jupyter calls `repr` on the value.

I would just add another line:
```
print(exa)
```